packages/bundler-plugin-core/src/index.ts

```diff
@@ -371,6 +399,18 @@ export function createRollupReleaseInjectionHooks(injectionCode: string) {
   };
 }
 
+export function createRollupBundleSizeOptimizationHooks(values: SentrySDKBuildFlags) {
+  return {
+    renderChunk(code: string) {
+      return replaceBooleanFlagsInCode(code, values);
+    },
+
+    transform(code: string) {
+      return replaceBooleanFlagsInCode(code, values);
+    },
+  };
```

Comment:
Why do we need to call the replace function in two hooks? Shouldn't `transform` be enough?

Reply:
Tbh I just copied this behaviour from the rollup replace Plugin 😅 they to it that way there. But I can just remove the chunk hook!